CorsixTH/CMakeLists.txt

```diff
@@ -14,6 +14,17 @@ elseif(MSVC)
   add_definitions(/D "_BIND_TO_CURRENT_VCLIBS_VERSION=1")
 endif()
 
+if(MSVC)
+  set(CORSIX_TH_DATADIR CorsixTH)
+  set(CORSIX_TH_INTERPRETER_PATH CorsixTH.lua)
+elseif(APPLE)
+  set(CORSIX_TH_DATADIR ${FULL_PRODUCT_NAME}/Contents/Resources/)
+  set(CORSIX_TH_INTERPRETER_PATH ${CORSIX_TH_DATADIR}/CorsixTH.lua)
```

Comment:
This is set to "/Contents/MacOS/CorsixTH.lua" which is correct relative to the app folder, but the working directory of an app is $home.

Reply:
This should be correct at install time, not build time. The initial slash is going to be a problem - otherwise do paths get evaluated relative to the app on OSX?